### PR TITLE
So let's discuss something else (Followup).

### PR DESCRIPTION
**Pull request by ghost**
This PR has a few config changes, mainly changing the xp formula to "x+1" and removing tomes from dropping entirely, thus making overlevelling utterly impossible. Instead, focus will now be shifted to picking your gear carefully (stat boosting relics)

The regret orbs will also drop less often to reduce potential clutter.

Something to discuss is whether we should make levelling a more lenient and add a 1/2 multiplier to x, making the level requirement grow twice slower. This might go in line more with the default configuration, which requires 25 levels at level ~50-70, but do we really need more levels with the relics?

### CODE REVIEW

**Comment by HyperSoop** [comment]
Oh, also remember to do a level wipe after (if) this gets merged

**Comment by Gaming32** [comment]
I think we should cap the level at a certain point. Maybe like 100 levels.

**Comment by HyperSoop** [comment]
How would we do that tho? (I mean, having to get 100 levels per level at level 100 might be an effective cap anyway)

**Comment by Gaming32** [comment]
It's not a hard cap though. The reason people don't want PlayerEx is because it's really grindy, and not capping it just makes it so that you can continue to grind.

**Comment by HyperSoop** [comment]
If you just make it linear, it'd be impossible to grind up too many levels and and grinding past level ~50 would be pretty much needless, so the grinders can still compete with each other for the higher number, but people uninterested in grinding wouldn't have to grind too much to keep up because the level difference between a grinder and a non-grinder would be at most 20 levels which would still be able to be outplayed with skill and strategy.

It might be useful to add an exponential increment though to solidify this concept.

**Comment by HyperSoop** [comment]
"x+x^(0.0001x^2)" might be good.

**Comment by HyperSoop** [comment]
With that, the difference between level 100 and 150 would be 80,000 levels in value per level, so it's an effective cap.

**Comment by Gaming32** [comment]
I think this formula is good. Probably what will end up happening, though, is a manual merge after the 1.19.2 update.

File: config/yosbr/config/relicex.json
Comment (EmmaTheGiant) [inline config/yosbr/config/relicex.json:10]: It is already too easy to get decent immortal gear. I got good relics within 15 minutes of playing.
We shouldn't change `"mobsDropLootChance"` and we should lower `"mobDropIsRelicChance"`.

File: config/yosbr/config/relicex.json
Comment (EmmaTheGiant) [inline config/yosbr/config/relicex.json:14]: This is something that should be left up to a vote.
I like the change, seeing as there are other methods of getting tomes, but they are still rare enough that the community should decide.

**Review by EmmaTheGiant — CHANGES_REQUESTED** [review]

File: config/yosbr/config/relicex.json
Comment (HyperSoop) [reply config/yosbr/config/relicex.json:10]: It's harder if you're starting at a low level, if you have high luckiness already you're gonna have an easy time getting a lot of relics. With the current config the base chance is basically 1.5% per mob? i do agree though, I'll make the relic chance lower.

File: config/yosbr/config/relicex.json
Comment (HyperSoop) [reply config/yosbr/config/relicex.json:14]: The point is to make tomes not drop at all from any source, if we're making levelling exponentially harder at higher levels we should not have the possibility of one getting essentially thousands of XP levels from a single mob drop.

**Comment by HyperSoop** [comment]
https://github.com/CleverNucleus/PlayerEx/wiki/FAQ#reverting-melee-crit-behaviour-to-vanilla this should be done

**Review by EmmaTheGiant — APPROVED** [review]

**Comment by HyperSoop** [comment]
I'm pretty sure it shouldn't require any other changes

**Comment by HyperSoop** [comment]
@Gaming32 i summon you

**Comment by Gaming32** [comment]
Lgtm ngl

**Comment by Gaming32** [comment]
Will merg later

**Pull request merged**
